src/aspire/basis/fb_2d.py

```diff
@@ -29,13 +29,16 @@ def __init__(self, size, ell_max=None, dtype=np.float32):
         Initialize an object for the 2D Fourier-Bessel basis class
 
         :param size: The size of the vectors for which to define the basis.
+            May be a 2-tuple or an integer, in which case a square basis is assumed.
             Currently only square images are supported.
         :ell_max: The maximum order ell of the basis elements. If no input
             (= None), it will be set to np.Inf and the basis includes all
             ell such that the resulting basis vectors are concentrated
             below the Nyquist frequency (default Inf).
         """
 
+        if isinstance(size, int):
```

Comment:
Maybe to cover these in testing we should initialize with an integer `size` in each basis test.